source/common/formatter/substitution_formatter.cc

```diff
@@ -1424,6 +1424,24 @@ const StreamInfoFormatter::FieldExtractorLookupTbl& StreamInfoFormatter::getKnow
                                                          ",");
                                   });
                             }}},
+                          {"DOWNSTREAM_PEER_DNS_SANS",
```

Comment:
The other SAN formatters use `_SAN` not `_SANS` (plural). The plural is more correct, but maybe we should keep them consistent?

Reply:
done